src/jormungandr/testing/jormungandr-integration-tests/src/networking/p2p/connections.rs

```diff
@@ -108,6 +115,91 @@ pub fn node_trust_itself() {
         .unwrap();
 }
 
+#[ignore]
+#[test]
+fn gateway_gossip() {
```

Comment:
hm, i don't think `cross_version` module is not a good place for this test method. Maybe it should be put into more meaningful module? like `gateway`?